.env.sample

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://root:root@localhost:5432/db"
```

Uma opção para ficar mais organizado o uso do env é criar uma variável para cada componente da URL. Por exemplo:
```suggestion
DB_HOST=localhost
DB_PORT=5432
DB_USER=root
DB_PASSWORD=root
DB_NAME=db
```
No contexto do uso do ORM a URL única é mais utilizada do que a divisão das variáveis, pois nesse cenário você montaria a url diretamente no código e não no env.